From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: James Shkolnik <jshkolnik@gmail.com>
Date: Sat, 27 Jun 2026 00:00:00 +0000
Subject: orangepi-zero3w: set model string + correct board compatible

Set a correct model and a board-specific compatible so
/proc/device-tree/{model,compatible} and any DT-based board detection are right.
---
 arch/arm64/boot/dts/allwinner/sun60i-a733-orangepi-zero3w.dts | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/allwinner/sun60i-a733-orangepi-zero3w.dts b/arch/arm64/boot/dts/allwinner/sun60i-a733-orangepi-zero3w.dts
index 111111111111..222222222222 100644
--- a/arch/arm64/boot/dts/allwinner/sun60i-a733-orangepi-zero3w.dts
+++ b/arch/arm64/boot/dts/allwinner/sun60i-a733-orangepi-zero3w.dts
@@ -6,7 +6,8 @@
 //#define TYPEC_DP 1
 /{
 	board = "A733";
-	compatible = "xunlong,orangepi-4-pro", "arm,sun60iw2p1";
+	model = "Orange Pi Zero 3W";
+	compatible = "xunlong,orangepi-zero3w", "arm,sun60iw2p1";
 	aliases {
 		arisc-config = &arisc_config;
 		pmu0 = &pmu0;
-- 
Armbian

